bin/deployer.cpp

```diff
@@ -238,13 +239,6 @@ int main(int argc, char** argv)
 #ifdef  ORO_BUILD_RTALLOC
     if (0 != rtMem)
     {
-        std::cout << "TLSF bytes allocated=" << memSize
-                  << " overhead=" << (memSize - freeMem)
-                  << " max-used=" << get_max_size(rtMem)
-                  << " currently-used=" << get_used_size(rtMem)
-                  << " still-allocated=" << (get_used_size(rtMem) - (memSize - freeMem))
-                  << "\n";
-
```

Comment:
I removed those because the same output has already been produces earlier using the RTT Logger.